### PR TITLE
Internal template

### PR DESCRIPTION
**Pull request by matharden**
This is mainly a better organisation of Sass files. I.e. each file is has single concern. 

It uses a new rebased version of template dependancies ([mojular](https://github.com/ministryofjustice/mojular/tree/internal) & [govuk-template](https://github.com/ministryofjustice/govuk-template)) and matches their directory structure.

### CODE REVIEW

**Comment by SteveMarshall** [comment]
Looks good to me, @matharden, but “Better organised Mojular styles” is a bit of a clunky commit message :disappointed:

File: mtp_cashbook/assets-src/stylesheets/components/_subheader.scss [hunk 1]
```diff
@@ -1,29 +0,0 @@
-.main-content {
-  padding-top: 0;
-}
-
-.global-subheader {
-  padding-top: 12px;
-  padding-bottom: 12px;
-  background: $grey-3-4;
```
Comment (teneightfive) [inline mtp_cashbook/assets-src/stylesheets/components/_subheader.scss]: Can we use a colour from the existing [govuk colour palette](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss)?

File: mtp_cashbook/assets-src/stylesheets/elements/_subheader.scss [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.global-subheader {
+  padding-top: 12px;
+  padding-bottom: 12px;
+  background: $grey-3-4;
```
Comment (teneightfive) [inline mtp_cashbook/assets-src/stylesheets/elements/_subheader.scss]: Can we use a colour from the existing [govuk colour palette](https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_colours.scss)?

**Comment by SteveMarshall** [comment]
Is this ready to be merged, @matharden?

**Pull request merged**